styles/styles.css

```diff
@@ -409,8 +455,197 @@ footer {
   }
 }
 
+@media screen and (min-width: 200px) and (max-width: 300px) {
+  /* WORKS SECTION */
+
+  .works-section {
+    margin-top: 10rem;
+  }
+
+  /* ABOUT MYSELF SECTION */
+
+  .about-myself {
+    margin-top: 3.5rem;
+  }
+}
+
 @media screen and (min-width: 376px) {
+  /* HEADER */
+
+  header {
+    background-image: url('../images/Header-llustration-desktop.svg');
+    background-size: cover;
+    background-position: center;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  /* HEADER */
+
+  header {
+    display: grid;
+    grid-template-areas:
+      'nav nav nav nav'
+      'hl hl hl hl'
+      'hl hl hl hl'
+      'hl hl hl hl';
+  }
+
+  nav {
+    grid-area: nav;
+  }
+
+  .mobile-ul {
+    display: none;
+  }
+
+  .desktop-ul {
+    display: flex;
+    justify-content: center;
+  }
+
+  /* HEADLINE */
+
+  .headline > h1 {
+    font-size: 48px;
+  }
+
+  /* RECENT WORKS SECTION */
+
+  .works-section {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .projects {
+    display: grid;
+    grid-template-columns: repeat(2, auto);
+  }
+
+  /* ABOUT ME SECTION */
+
+  .about-myself {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .about-myself-header {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    height: 17rem;
+    margin-bottom: 3rem;
+  }
+
+  .about-myself-header > p {
+    width: 77%;
+  }
+
+  .ab-me-cards-container {
+    display: grid;
+    grid-template-columns: repeat(3, 33.33%);
+    margin: 0 3rem;
+  }
+
+  .about-me-card {
+    width: 95%;
+  }
```

Comment:
- [ ] Your language card does not have the same alignment as the rest due to the different amount of items in its list. Try changing the `justify-content` and using margins to reposition the contents inside this card so they can have the same alignment as the Figma template.
 -Your design:
<img width="1265" alt="image" src="https://user-images.githubusercontent.com/61919617/186709131-7d8423de-5849-4e29-9c37-58993ceb54ce.png">
-Figma template:
<img width="1159" alt="image" src="https://user-images.githubusercontent.com/61919617/186709230-b8696e45-bac9-4f11-8559-c885f80211c1.png">